src/main/java/io/antmedia/AntMediaApplicationAdapter.java

```diff
@@ -653,6 +655,10 @@ public VideoServiceEndpoint getVideoServiceEndPoint(String id) {
 		return null;
 	}
 
+	public void setPreviewPathForVod(String path){
```

Comment:
Setting preview path for VoD like this in a multi-thread and async environment will likely cause wrong mapping.